packages/app-store/routing-forms/components/FormActions.tsx

```diff
@@ -230,7 +230,7 @@ function Dialogs({
   });
   return (
     <div id="form-dialogs">
-      <EmbedDialog />
+      <EmbedDialog embedFor="routing-form" />
```

Comment:
Let's avoid context aware components. 

```suggestion
      <EmbedDialog disabledTypes={["email"]} />
```

Reply:
I am not very sure. I seem to like my approach better. It's like having a HOC e.g. EventTypeEmbed and RoutingFormEmbed. Whenever I need to make a CTA for RoutingForm embed I can use the HOC. 
Also, it's not like Embed Routing Form would support email embed, it should never be enabled for Routing Form embed. So, providing embed this context ensures this.

Reply:
Still is very context dependent. Lots of conditions to be aware of. Generic options optimize for legibility as well.

Reply:
> Whenever I need to make a CTA for RoutingForm embed I can use the HOC.

You just add a generic CTA prop then. See where I'm going with the generic approach? Components should not care about business logic.

Reply:
Cool. Thanks. I will give it a try and see how it look🙏

Reply:
Do you have some reading material on this? Would love to learn more 🙏

Reply:
Thanks to you for the patience @hariombalhara I'm just trying to keep the code friendly for everyone. Not only those with business context in mind.